{"name": "test_change_not_auth_user_fail", "status": "passed", "description": "Изменение данных неавторизованного пользователя", "start": 1720287838079, "stop": 1720287838262, "uuid": "ebb111f0-f43a-4bb2-988b-bfc5f5f59b1a", "historyId": "573686f6ec4567847eb8e31d3d321bed", "testCaseId": "573686f6ec4567847eb8e31d3d321bed", "fullName": "tests.test_change_user_data.TestChangeUserData#test_change_not_auth_user_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_change_user_data"}, {"name": "subSuite", "value": "TestChangeUserData"}, {"name": "host", "value": "DESKTOP-LU3QF62"}, {"name": "thread", "value": "22836-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_change_user_data"}]}